src/test/java/atas/logic/commands/memo/AddNoteCommandTest.java

```diff
@@ -0,0 +1,290 @@
+
+        @Override
+        public ReadOnlyStudentList getStudentList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public boolean hasStudent(Student student) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void deleteStudent(Student target, Index id) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setStudent(Student target, Student editedStudent) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public Path getSessionListFilePath() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setSessionListFilePath(Path sessionListFilePath) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void resetSessionList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public SessionList getSessionList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public boolean hasSession(Session session) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void deleteSession(Session target, Index id) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void addSession(Session session) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void enterSession(Index sessionIndex) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ObservableList<Attributes> getFilteredAttributesList() {
+            return null;
+        }
+
+        @Override
+        public void setSession(Session target, Session editedSession) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void updateParticipationBySessionName(SessionName sessionName, IndexRange indexRange) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void updatePresenceBySessionName(SessionName sessionName, IndexRange indexRange) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ObservableList<Student> getFilteredStudentList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ObservableList<Session> getFilteredSessionList() {
+            return null;
```

Comment:
Perhaps throw an AssertionError here instead?

Reply:
Good suggestion! Will make the necessary changes 👍